charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+description: kube-prometheus-stack collects Kubernetes manifests, Grafana dashboards, and Prometheus rules combined with documentation and scripts to provide easy to operate end-to-end Kubernetes cluster monitoring with Prometheus using the Prometheus Operator.
+icon: https://raw.githubusercontent.com/prometheus/prometheus.github.io/master/assets/prometheus_logo-cb55bb5c346.png
+engine: gotpl
+maintainers:
+  - name: vsliouniaev
+  - name: bismarck
+  - name: gianrubio
+    email: gianrubio@gmail.com
+  - name: gkarthiks
+    email: github.gkarthiks@gmail.com
```

Comment:
Would you mind adding yourself as a maintainer ?

Reply:
@gianrubio I guess you are asking @scottrigby right?

Reply:
> @gianrubio I guess you are asking @scottrigby right?

Exactly!

Reply:
Added